<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFirstRun</class>
 <widget class="QMainWindow" name="frmFirstRun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>715</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>715</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SmartClass | Nintersoft</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/logos/Logo.png</normaloff>:/images/logos/Logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#titleBar{
	background-color: #FFF;
	border-top: 1px solid orange;
	border-left: 1px solid orange;
	border-right: 1px solid orange;
	color: #000;
}

QWidget#centralWidget, QStatusBar{
	background-color: #FFF;
	border-left: 1px solid orange;
	border-right: 1px solid orange;
}

QStatusBar{
	border-bottom: 1px solid orange;
}

/*QTabWidget::pane {
    border-top: 1px solid orange;
}*/

QSlider::handle:horizontal {
    background: orange;
}

/* Vertical and horizontal line customization */

QFrame[frameShape=&quot;4&quot;]
{
    color: orange;
}

QFrame[frameShape=&quot;5&quot;]
{
    color: orange;
}

/* QTabWidget elements customization */

QTabWidget::pane {
/*	border-top: 1px solid orange;*/
	background: white;
}

QTabWidget::tab-bar {
    left: 5px;
}

QTabBar::tab {
	background: white;
    min-width: 120px;
    min-height: 20px;
    padding: 2px;
	border-top-left-radius: 5px;
	border-top-right-radius: 5px;
	margin-top: 6px;
}

QTabBar::tab:selected, QTabBar::tab:hover {
	background: white;
	border-top: 1px solid orange;
	border-right: 1px solid orange;
	border-left: 1px solid orange;
}

QTabBar::tab:selected {
	background: white;
	border-top: 2px solid orange;
	border-right: 2px solid orange;
	border-left: 2px solid orange;
}

QTabBar::tab:!selected {
    margin-bottom: 3px;
}

QTabBar QToolButton{
	background-color: white;
	border: none;
	color: black;
}

QTabBar QToolButton::hover{
	background-color: darkorange;
}

QTabBar QToolButton::pressed{
	background-color: orange;
}

QTabBar QToolButton::!enabled{
	background-color: orange;
}

QWidget#tabFile, QWidget#tabManagement, QWidget#tabTools, QWidget#tabHelp, QWidget#tabFileLayout, QWidget#tabManagementLayout, QWidget#tabToolsLayout, QWidget#tabHelpLayout, QWidget#screenManagerLayout{
	background: white;
}

QLineEdit{
	border: 1px solid orange;
	min-height: 20px;
}

QPlainTextEdit{
	border: 1px solid orange;
	min-height: 40px;
}

QPushButton{
	background-color: darkorange;
	border: 5px solid darkorange;
	color: white;
}

QPushButton::hover{
	background-color: orange;
	border: 5px solid orange;
}

QPushButton::pressed{
	background-color: darkorange;
	border: 5px solid darkorange;
}

QPushButton::!enabled{
	background-color: orange;
	border: 5px solid orange;
}

/*
	QScroll Area
*/

QScrollArea{
	border: none;
}

QScrollArea#scrollManager{
	border: 1px solid orange;
	border-radius: 5px;
}

/* QScrollBar Horizontal*/

QScrollBar:horizontal {
    background: lightgray;
    height: 10px;
    margin: 0px 40px 0 0px;
	border-radius: 5px;
}

QScrollBar::handle:horizontal {
    background: orange;
	border-radius: 5px;
    min-width: 20px;
}

QScrollBar::add-line:horizontal {
    background: blue;
    width: 20px;
    subcontrol-position: right;
    subcontrol-origin: margin;
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
}

QScrollBar::sub-line:horizontal {
    background: red;
    width: 20px;
    subcontrol-position: top right;
    subcontrol-origin: margin;
    position: absolute;
    right: 20px;
	border-top-left-radius: 5px;
	border-bottom-left-radius: 5px;
}

QScrollBar::sub-line:horizontal:hover {
    background: darkred;
}

QScrollBar::sub-line:horizontal:pressed {
    background: red;
}

QScrollBar::add-line:horizontal:hover {
    background: darkblue;
}

QScrollBar::add-line:horizontal:pressed {
    background: blue;
}

QScrollBar:left-arrow:horizontal{
	image: url(:/images/scrollbar/slide_left.PNG);
    width: 10px;
    height: 10px;
}

QScrollBar::right-arrow:horizontal {
	image: url(:/images/scrollbar/slide_right.PNG);
    width: 10px;
    height: 10px;
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: none;
}

/* QScrollBar Vertical*/

QScrollBar:vertical {
    background: lightgray;
    width: 10px;
    margin: 0px 0px 40px 0px;
	border-radius: 5px;
}

QScrollBar::handle:vertical {
    background: orange;
	border-radius: 5px;
    min-width: 20px;
}

QScrollBar::add-line:vertical {
    background: blue;
    height: 20px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
	border-bottom-right-radius: 5px;
	border-bottom-left-radius: 5px;
}

QScrollBar::sub-line:vertical {
    background: red;
    height: 20px;
    subcontrol-position: bottom right;
    subcontrol-origin: margin;
    position: absolute;
    bottom: 20px;
	border-top-right-radius: 5px;
	border-top-left-radius: 5px;
}

QScrollBar::sub-line:vertical:hover {
    background: darkred;
}

QScrollBar::sub-line:vertical:pressed {
    background: red;
}

QScrollBar::add-line:vertical:hover {
    background: darkblue;
}

QScrollBar::add-line:vertical:pressed {
    background: blue;
}

QScrollBar:up-arrow:vertical{
	image: url(:/images/scrollbar/slide_up.PNG);
	width: 10px;
    height: 10px;
}

 QScrollBar::down-arrow:vertical{
	image: url(:/images/scrollbar/slide_down.PNG);
	width: 10px;
    height: 10px;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: none;
}

/*QComboBox*/

QComboBox {
    border: 1px solid orange;
	background: white;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
	background: white;
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: white;
}

QComboBox:on {
    padding-top: 3px;
    padding-left: 4px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: orange;
    border-left-style: solid;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox::drop-down:hover {
    background: orange;
}

QComboBox::down-arrow {
    image: url(&quot;:/images/buttons/Down_Arrow.PNG&quot;);
	width: 10px;
	height: 10px;
}

QComboBox::down-arrow:on {
    top: 1px;
    left: 1px;
}

QComboBox QAbstractItemView {
    border: 1px solid orange;
    selection-background-color: orange;
    border-top-right-radius: 3px;
    border-bottom-right-radius: 3px;
}

QComboBox QAbstractItemView:hover {
	color: orange;
}

/*
 * QToolButton
 */

QToolButton {
	background: transparent;
	border: 1px solid transparent;
	color: black;
}

QToolButton::hover {
	background: darkorange;
	border: 1px solid darkorange;
	color: white;
}

QToolButton::pressed {
	background: orange;
	border: 1px solid orange;
	color: white;
}

/* QSpinBox  QDateEdit*/

QSpinBox, QDoubleSpinBox, QDateEdit, QDateTimeEdit {
	border: 1px solid orange;
	background: white;
    padding-right: 15px; /* make room for the arrows */
	border-radius: 5px;
}

QSpinBox::up-button:disabled,  QSpinBox::up-button:off,
QSpinBox::down-button:disabled,  QSpinBox::down-button:off,
QDoubleSpinBox::up-button:disabled,  QDoubleSpinBox::up-button:off,
QDoubleSpinBox::down-button:disabled,  QDoubleSpinBox::down-button:off,
QDateEdit::up-button:disabled,  QDateEdit::up-button:off,
QDateEdit::down-button:disabled,  QDateEdit::down-button:off,
QDateTimeEdit::up-button:disabled,  QDateTimeEdit::up-button:off,
QDateTimeEdit::down-button:disabled,  QDateTimeEdit::down-button:off{
	background: orange;
}

QSpinBox::up-button, QDoubleSpinBox::up-button, QDateEdit::up-button, QDateTimeEdit::up-button {
    subcontrol-origin: border;
    subcontrol-position: top right;

    width: 16px;
	border: 1px solid orange;
	border-top-right-radius: 5px;
}

QSpinBox::up-button:hover, QDoubleSpinBox::up-button:hover, QDateEdit::up-button:hover, QDateTimeEdit::up-button:hover {
	background: darkorange;
}

QSpinBox::up-button:pressed, QDoubleSpinBox::up-button:pressed, QDateEdit::up-button:pressed, QDateTimeEdit::up-button:pressed {
	background: orange;
}

QSpinBox::up-arrow, QDoubleSpinBox::up-arrow, QDateEdit::up-arrow, QDateTimeEdit::up-arrow {
    border-image: url(:/images/buttons/Up_Arrow.PNG) 1;
    width: 7px;
    height: 7px;
}

QSpinBox::up-arrow:disabled,
QSpinBox::up-arrow:off,
QDoubleSpinBox::up-arrow:disabled,
QDoubleSpinBox::up-arrow:off,
QDateEdit::up-arrow:disabled,
QDateEdit::up-arrow:off,
QDateTimeEdit::up-arrow:disabled,
QDateTimeEdit::up-arrow:off {
   background: transparent;
}

QSpinBox::down-button,
QDoubleSpinBox::down-button,
QDateEdit::down-button,
QDateTimeEdit::down-button {
    subcontrol-origin: border;
    subcontrol-position: bottom right;

    width: 16px;
	border: 1px solid orange;
	border-bottom-right-radius: 5px;
}

QSpinBox::down-button:hover,
QDoubleSpinBox::down-button:hover,
QDateEdit::down-button:hover,
QDateTimeEdit::down-button:hover {
	background: darkorange;
}

QSpinBox::down-button:pressed,
QDoubleSpinBox::down-button:pressed,
QDateEdit::down-button:pressed,
QDateTimeEdit::down-button:pressed {
	background: orange;
}

QSpinBox::down-arrow,
QDoubleSpinBox::down-arrow,
QDateEdit::down-arrow,
QDateTimeEdit::down-arrow {
    border-image: url(:/images/buttons/Down_Arrow.PNG) 1;
    width: 7px;
    height: 7px;
}

QSpinBox::down-arrow:disabled,
QSpinBox::down-arrow:off,
QDoubleSpinBox::down-arrow:disabled,
QDoubleSpinBox::down-arrow:off,
QDateEdit::down-arrow:disabled,
QDateEdit::down-arrow:off,
QDateTimeEdit::down-arrow:disabled,
QDateTimeEdit::down-arrow:off {
   background: transparent;
}

/*QCheckBox*/

QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 13px;
    height: 13px;
}

QCheckBox::indicator:unchecked {
    border: 1px solid orange;
}

QCheckBox::indicator:unchecked:hover {
    border: 2px solid darkorange;
}

QCheckBox::indicator:unchecked:pressed {
    border: 2px solid orange;
}

QCheckBox::indicator:checked {
    border: 1px solid orange;
	image: url(:/images/buttons/Close.PNG);
}

QCheckBox::indicator:checked:hover {
    border: 2px solid darkorange;
	image: url(:/images/buttons/Close.PNG);
}

QCheckBox::indicator:checked:pressed {
    border: 2px solid orange;
	image: url(:/images/buttons/Close.PNG);
}

QCheckBox::indicator:indeterminate:hover {
    border: 2px solid darkorange;
}

QCheckBox::indicator:indeterminate:pressed {
    border: 2px solid orange;
}

/*
 * QTableWidget
 */

QTableWidget{
	border: 1px solid orange;
}

/*
 * QGroupBox
 */

QGroupBox {
    border: 1px solid orange;
    border-radius: 2px;
    margin-top: 10px;
    padding-top: 10px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top middle; /* position at the top center */
    padding: 5px;
    background-color: white;
	border: 1px solid orange;
	border-radius: 2px;
}

/*
 * QListWidget
 */

QListWidget {
    border: 1px solid orange;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="TitleBar" name="titleBar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>10</number>
      </property>
      <item row="8" column="0" rowspan="6">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="5">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="btSearchCompanyLogo">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QLabel" name="lblWelcome">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>You're welcome to SmartClass</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="edtCompanyLogoPath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QLabel" name="lblFormDescription">
        <property name="text">
         <string>We still have to complete some steps in order to use the program.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="5">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="3">
       <widget class="QPushButton" name="btRemoveLogo">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="3">
       <widget class="QLabel" name="lblCompanyLogo">
        <property name="text">
         <string>Company logo (use the button to find the image):</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="3">
       <widget class="QLineEdit" name="edtCompanyName">
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="3">
       <widget class="QGroupBox" name="grpDatabaseSchema">
        <property name="title">
         <string>Database Schema</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QRadioButton" name="rbSQLite">
           <property name="text">
            <string>SQLite database (local file)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="rbMySQL">
           <property name="text">
            <string>MySQL database (remote database without file schema)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLineEdit" name="edtDatabasePrefix"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblDatabasePrefix">
           <property name="text">
            <string>Table prefix (only if you have multiple SmartClass connections on a shared database - leave empty otherwise)</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLabel" name="lblCompanyName">
        <property name="text">
         <string>Company name:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1" colspan="3">
       <widget class="QPushButton" name="btSaveData">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1" colspan="3">
       <widget class="QGroupBox" name="grpMySQLSettings">
        <property name="title">
         <string>MySQL Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="lblPortNumber">
           <property name="text">
            <string>Port:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblDatabaseName">
           <property name="text">
            <string>Database name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblHostName">
           <property name="text">
            <string>Host:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lblUsername">
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lblPassWord">
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="edtHost"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="edtDatabase"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="edtUsername"/>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="edtPassword"/>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="edtPortNumber">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="9" column="1" colspan="3">
       <widget class="QGroupBox" name="grpLanguage">
        <property name="title">
         <string>Language</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="lblLanguage">
           <property name="text">
            <string>Language:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbLanguage">
           <item>
            <property name="text">
             <string>English (UK)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Portuguese (BR)</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBar</class>
   <extends>QWidget</extends>
   <header>titlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtCompanyName</tabstop>
  <tabstop>edtCompanyLogoPath</tabstop>
  <tabstop>btSearchCompanyLogo</tabstop>
  <tabstop>btRemoveLogo</tabstop>
  <tabstop>cbLanguage</tabstop>
  <tabstop>rbSQLite</tabstop>
  <tabstop>rbMySQL</tabstop>
  <tabstop>edtHost</tabstop>
  <tabstop>edtPortNumber</tabstop>
  <tabstop>edtDatabase</tabstop>
  <tabstop>edtUsername</tabstop>
  <tabstop>edtPassword</tabstop>
  <tabstop>btSaveData</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
